iset.mm

```diff
@@ -70783,6 +70789,15 @@ to define the cut as the lower set only (as Metamath Proof Explorer does),
 but we define the cut as a pair of both the lower and upper sets, as [HoTT]
 does. There are also differences in how we handle order and replacing
 "not equal to zero" with "apart from zero".
+
+When working without excluded middle, there are several possible definitions
```

Comment:
Should we add "nor the axiom of choice" or "nor choice principles" or "nor the axiom of choice or its weaker forms" ? Or not ? You be the judge.

Reply:
My initial draft was just "When working constructively" - maybe that is better. It hardly seems like the place to go deeply into this topic - the HoTT book mentions both countable choice and power sets in this context, and the BauerHanson paper has a little bit of this sort of thing too.

Reply:
"When working constructively" is probably better indeed.

Reply:
Changed.